source/accessibility-statement.html.md.erb

```diff
@@ -0,0 +1,96 @@
+---
+title: Accessibility
+weight: 999
```

Comment:
This property shouldn't be needed, as you're hiding the page from the navigation on the next line.

I think the [Tech docs template documentation](https://tdt-documentation.london.cloudapps.digital/configure_project/frontmatter/#required-frontmatter-settings) is misleading here ("In a multipage documentation site, you must include the weight frontmatter setting"). The [code](https://github.com/alphagov/tech-docs-gem/blob/aa7ea93f144b777e2b654bcd8a21016d4420bd7d/lib/govuk_tech_docs/table_of_contents/helpers.rb#L26) looks like it will set `weight` to a default value if one isn't provided here, so it's not going to break the building of the docs.

Reply:
thanks @ChrisBAshton - this is what we have for other docs sites so going to keep it in here for now to get this up and will flag for the next iteration of work on the Tech Docs Template so it can be cleared of anything misleading